### DISCUSSION TITLE
Favorite games

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#636] [General]
What are you playing guys?
Long winter is on its way, need some suggestions.

**Comment by MirkosMangle** [comment]
We love playing board games in our family.

**Comment by ellielo** [comment]
Found a new one, called "Code names" . Great one!

**Comment by hernessteve** [comment]
Thanks a lot. Will check this one out. Now have much more time since i found services make my homework done for me https://academized.com/do-my-homework .  Now i can start playing games again, watch movies and episodes in the evening, have fun with the friends. I don't need to spend all my free time in front of my books and computer.

**Comment by Gloria109** [comment]
I love playing board games

**Comment by mariawotters** [comment]
I spend a lot of time at home, especially in winter, and I’m always looking for new ways to entertain myself. The one thing that never changes is playing GTA 5 Online with my friends and poker, thanks to https://www.thekingpluscasino.net/ . I’m thinking about starting to work out, not just playing games but also training my body physically. I’ll probably start with fitness and maybe even yoga.

**Comment by Jonathan19988** [comment]
I like play CS2)
